src/content/roadmap/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Ethereum roadmap
+description: The path to more scalability, security and sustainability for Ethereum.
+lang: en
+template: roadmap
+image: ../../assets/upgrades/merge.png
+alt: "Ethereum roadmap"
+summaryPoints:
+  - Learn about how Ethereum will evolve into the future
+  - Read why Ethereum is still being worked on
+  - Understand how Ethereum is setting up for long term success
```

Comment:
Same comment as on account abstraction. We should be summarising, not telling users the objectives of the page.

cc: @konopkja

Reply:
ah - yea this is placeholder frontmatter that should be deleted, the bullets are not actually used on the page.